rusoto/core/Cargo.toml

```
@@ -28,7 +28,7 @@ crc32fast = "1.2"
 futures = "0.3"
 http = "0.2"
 hyper = { version = "0.14", features = ["client", "http1", "http2", "tcp"] }
-hyper-rustls = { version = "0.22", optional = true, default-dependencies = false }
+hyper-rustls = { version = "0.23", optional = true, default-dependencies = false, features = [ "http2" ] }
```

This likely needs to be:

```suggestion
hyper-rustls = { version = "0.23", optional = true, default-dependencies = false, features = [ "http1", "http2", "tls12", "logging" ] }
```

(What I think we want is upstream's `default-features` minus `native-tokio`, because that is selected by our `rustls` or `rustls-webpki` features. These are all new default features in 0.23.)

I'll add this change to the merge.